Resources/config/services.php

```diff
@@ -54,6 +59,22 @@
 
     $services->alias(RefreshTokenManagerInterface::class, 'gesdinet.jwtrefreshtoken.refresh_token_manager');
 
+    $services->set('gesdinet.jwtrefreshtoken.request.extractor.chain')
+        ->class(ChainExtractor::class)
+        ->public();
+
+    $services->alias(ExtractorInterface::class, 'gesdinet.jwtrefreshtoken.request.extractor.chain');
+
+    $services->set('gesdinet.jwtrefreshtoken.request.extractor.request_body')
+        ->class(RequestBodyExtractor::class)
+        ->public()
```

Comment:
I think we don't need to have the extractors in public, the chain is enough.

Reply:
Yeah, you’re probably right.  That was most likely just excessive copy/paste.

Reply:
It's just a detail, don't listen me 😄 
It's a great evolution in any case, good job !